Documentation/Configuration/Typo3ConfVars/SYS.rst

```diff
@@ -246,6 +246,22 @@ configurations.
     Commalist of file extensions perceived as media files by TYPO3.
     Must be written in lower case with no spaces between.
 
+..  _typo3ConfVars_sys_miscfile_ext:
+
+..  confval:: miscfile_ext
+    :name: globals-typo3-conf-vars-sys-miscfile-ext
+    :Path: $GLOBALS['TYPO3_CONF_VARS']['SYS']['miscfile_ext']
+    :type: text
+    :Default: 'gif,jpg,jpeg,bmp,png,pdf,svg,ai,mp3,wav,mp4,ogg,flac,opus,webm,youtube,vimeo'
```

Comment:
misc file ext is just "zip"
```suggestion
    :Default: 'zip'
```

Reply:
ah right, I wanted to look  if it has a default value and then forgot it.